test/realtimecontroller/DefaultRealtimeController.test.ts

```diff
@@ -538,6 +544,38 @@ describe('DefaultRealtimeController', () => {
       expect(callbackFired).to.be.true;
     });
 
+    it('will only unsubscribe from a given callback if provided', () => {
+      const rt: RealtimeController = new DefaultRealtimeController();
+      const sentAttendeeId = 'foo-attendee';
+      const sentVolume = 0.5;
+      const sentMuted = false;
+      const sentSignalStrength = 1;
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      const state: RealtimeState = ((rt as unknown) as any).state as RealtimeState;
+      state.volumeIndicatorCallbacks = {};
+      const callback = (
+        attendeeId: string,
+        volume: number | null,
+        muted: boolean | null,
+        signalStrength: number | null
+      ): void => {
+        expect(attendeeId).to.equal(sentAttendeeId);
+        expect(volume).to.equal(sentVolume);
+        expect(muted).to.equal(sentMuted);
+        expect(signalStrength).to.equal(sentSignalStrength);
+      };
+      rt.realtimeUpdateVolumeIndicator(
+        sentAttendeeId,
+        sentVolume,
+        sentMuted,
+        sentSignalStrength,
+        null
+      );
+      rt.realtimeSubscribeToVolumeIndicator(sentAttendeeId, callback);
```

Comment:
I think here the subscription needs to come first and then the update right?
```
rt.realtimeSubscribeToVolumeIndicator(sentAttendeeId, callback);
rt.realtimeUpdateVolumeIndicator(
  sentAttendeeId,
  sentVolume,
  sentMuted,
  sentSignalStrength,
  null
);
rt.realtimeUnsubscribeFromVolumeIndicator(sentAttendeeId, callback);
```

Reply:
Why do we need update? This test is just testing the unsubscribe method.

Reply:
Yes, but since the test has `realtimeUpdateVolumeIndicator` called before `realtimeSubscribeToVolumeIndicator` I thought so.

Reply:
Ha! I thought I had already removed that. I was confused. Fixed.